### DISCUSSION TITLE
Cannot deploy Airbyte to multiple namespaces

### DISCUSSION THREAD

**Discussion by col** [airbytehq/airbyte#47265] [Deployment]
I've like to have two deployments of Airbyte (you can think of these as prod and nonprod) within the same K8s cluster. This currently is not possible due to the `node-viewer` ClusterRole created by the Helm chart. As this is a cluster wide resource is conflicts when I try to deploy Airbyte to a second namespace. 

I believe this is an avoidable conflict but needs either:
- a configuration option, something like `.Values.serviceAccount.createClusterRole`
- a check to see if it already exists or not, something like `if not (lookup "rbac.authorization.k8s.io/v1" "ClusterRole" "" "node-viewer")`

**Comment by mkrawc** [comment]
Related issue: https://github.com/airbytehq/airbyte/issues/46401

[a comment by an automated account is not shown]
